aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -319,6 +322,14 @@ class AztecText : EditText, TextWatcher {
         return super.onKeyPreIme(keyCode, event)
     }
 
+    override fun onKeyUp(keyCode: Int, keyEvent: KeyEvent): Boolean {
+        if (formatToolbar.onKeyUp(keyCode, keyEvent)) {
```

Comment:
There is no guarantee that the client will call `setToolbar()` (or not pass `null`) so, we prolly should check it here. It might be simpler to just declare `formatToolbar` as `AztecToolbar?`.